unit-test/dirSize.js

```diff
@@ -1,19 +1,45 @@
-// @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
-import fastFolderSizeSync from 'fast-folder-size/sync.js'
+import { join, relative } from 'node:path'
+import { statSync } from 'node:fs'
 
-const buildDirs = ['android', 'chrome', 'chrome-mv3', 'firefox', 'integration', 'windows']
-// Lets sanity check build sizes, picking 512KB as a rather arbitrary limit
-// Higher for Chrome MV2 due to base64 encoding
-const quotas = [512000, 716800, 512000, 512000, 512000, 512000]
-const rootDir = 'build'
+// path helpers
+const ROOT = new URL('..', import.meta.url).pathname
+const BUILD = join(ROOT, 'build')
+const APPLE_BUILD = join(ROOT, 'Sources/ContentScopeScripts/dist')
 
-describe('Expect build size of', () => {
-    for (const build in buildDirs) {
-        const dir = `${rootDir}/${buildDirs[build]}`
-        const sizeLimit = quotas[build]
+const checks = {
+    android: [
+        { kind: 'maxFileSize', value: 512000, path: join(BUILD, 'android/contentScope.js') }
+    ],
+    chrome: [
+        { kind: 'maxFileSize', value: 716800, path: join(BUILD, 'chrome/inject.js') }
```

Comment:
We should preserve the previous comment, I'd prefer we make the whole file simpler not more complex unless you're foreseeing using this for the HTML too?

Reply:
yep it would be good to have a way to add more checks.

if you think this is 'too complex', we can revert and add the other checks else where 👍🏻

Reply:
The old version:

1: Only works for a directory, which no longer makes sense
2: didn’t support apple
3: used a 3rd party lib

so yes, it’s now more complex to solve those

Reply:
Nice missed that:

- Please remove the other lib :)
- Can we use a single int for the output size and make a 40% difference calculation for Chrome?

> if you think this is 'too complex', we can revert and add the other checks else where 👍🏻

Nah, if there's intent to do this soon that's fine by me; I just didn't want the if statement and 'kind' complexity if there wasn't the need for it.

Reply:
done 👍🏻